### DISCUSSION TITLE
blog/gf5tiw/

### DISCUSSION THREAD

**Discussion by giscus-comment** [wztlink1013/comment#58] [General]
# blog/gf5tiw/

第三方模块的两种存在形式：以js文件的形式存在，提供实现项目具体功能的API接口。比如使用jQuery，html中引入CDN超链接即可使用。以命令行工具形式存在，辅助项目开发。比如VUE-CLI工具。npm是用来管理模块的一个node工具，它会伴随着node的下载而一同下载。node生态的繁荣多益于

https://www.wztlink1013.com/blog/gf5tiw/

**Comment by wzutao** [comment]
模块化
